README.md

```diff
@@ -1,13 +1,17 @@
-# AWS IoT Device SDK C v4.0.0
+# AWS IoT Device SDK C v4_beta_deprecated
 
-**[Link to API documentation](https://docs.aws.amazon.com/freertos/latest/lib-ref/c-sdk/main/index.html)**
+**[Link to API documentation](https://docs.aws.amazon.com/freertos/latest/lib-ref/embedded-csdk/v4.0_beta_deprecated/lib-ref/index.html)**
 
```

Comment:
this link is deal, the readme should be updated along with the docs being available.

Reply:
this link goes live on wednesday.. it's the new link

Reply:
yes, so when does this PR gets merged? Its on wednesday not today, right?

Reply:
Why the link goes live Wednesday ? The v4_beta_deprecated is not getting released, is getting renamed ( which already has happened).

Reply:
I believe this PR should get merge ASAP to avoid any further confusion, we should not wait till Wednesday.

Reply:
yes^^ will do, Amit and I just chatted.  We will merge this asap and get that link live.

Reply:
The link does not seem to be live yet.